website_cookieconsent/views/assets.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <data>
+        <template id="assets_frontend" inherit_id="website.assets_frontend"
+                  name="Website cookie consent assets">
+            <xpath expr="." position="inside">
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/css-components/base.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/css-components/consent-modal.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/css-components/dark-scheme.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/css-components/light-scheme.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/css-components/preferences-modal.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/cookieconsent/cookieconsent.css"/>
+                <link rel="stylesheet"
+                      href="/website_cookieconsent/static/src/css/website_cookieconsent.css"/>
+                <link rel="stylesheet" href="/website_cookieconsent/static/src/js/iframemanager/css/iframemanager.css"/>    
+                <script src="/website_cookieconsent/static/src/js/cookieconsent/cookieconsent.js"></script>
+                                <script src="/website_cookieconsent/static/src/js/iframemanager/js/iframemanager.js"></script>  
```

Comment:
When including libraries, could you add the unminified ones instead? Odoo has [its own minifier](https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/ir/ir_qweb.py#L1568) and in this way we can unminify by going into debug=assets mode if we want to set some breakpoints or do debugging.

Reply:
@thomaspaulb I added this for the js libs.